cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -59,6 +59,7 @@
       cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'cron', 'fetch_external_resources'), notify:'dev+crontab@code.org'
       cronjob at:'30 17 * * *', do:deploy_dir('bin', 'cron', 'update_dotd')
       cronjob at:'0 13 * * *', do:deploy_dir('tools', 'scripts', 'brokenLinkChecker', 'brokenLinkChecker.js'), notify:'dev+crontab@code.org'
+      cronjob at:'30 15 * * 1-5', do:deploy_dir('bin', 'cron', 'deploy_to_test')
```

Comment:
That is correct (at least the intent). Feel welcome to suggest an alternate time, I see 7:30 AM as being sufficiently early to have the DTT finished before the 9:30 AM DOTD starts work (which often happens pre-9:30 AM). It is also late enough that I should be online to monitor hiccups in the early days.